spec/features/incoming_mail_handler_spec.rb

```diff
@@ -0,0 +1,79 @@
+describe "Handles incoming email" do
+  let(:proposal) { FactoryGirl.create(:proposal, :with_parallel_approvers) }
```

Comment:
Commented in the article about the "slow tests" argument no longer applying (as they are lazily loaded), but I hear you about the setting up dependencies explicitly. I try to use `let` judiciously – for simple data that is in fact shared by all of the tests – rather than using them to create every variable. That being said, I totally hear the Mystery Guest argument.